interface/mhtjuProducerCppWorker.h

```diff
@@ -0,0 +1,28 @@
+#ifndef PhysicsTools_NanoAODTools_mhtjuProducerCppWorker_h
+#define PhysicsTools_NanoAODTools_mhtjuProducerCppWorker_h
+
+#include <vector>
+#include <TTreeReaderValue.h>
+#include <TTreeReaderArray.h>
+#include "DataFormats/Math/interface/LorentzVector.h"
+
+class mhtjuProducerCppWorker {
+
+public:
+
+  mhtjuProducerCppWorker(){};
```

Comment:
no ";"
also, better initialize the pointers (to nullptr)
(this is an example, so it should be better written very cleanly)